### PR TITLE
Added 2 presentations #23

### PR DESCRIPTION
**Pull request by sumalaika**
Will add more presentations later

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -59,6 +59,11 @@ JanusGraph is a project under [The Linux
 Foundation](https://www.linux.com/blog/Linux-Foundation-welcomes-JanusGraph),
 and includes participants from Expero, Google, GRAKN.AI, Hortonworks, and IBM.
 
+## <a name="presentations"></a>Presentations
```
Comment (mbrukman) [inline index.md]: The `<a name="...">` is unnecessary because `## Presentations` will automatically create that anchor.

File: index.md [hunk 2]
```diff
@@ -59,6 +59,11 @@ JanusGraph is a project under [The Linux
...
+Here is a selection of Janusgraph presentations:
+* <a href="https://www.slideshare.net/ptgoetz/large-scale-graph-analytics-with-janusgraph">DataWorksJun2017: Large Scale Graph Analytics with JanusGraph</a>, P. Taylor Goetz, 2017.06.13
```
Comment (mbrukman) [inline index.md]: Please use Markdown syntax for links, rather thank HTML.

Comment (mbrukman) [inline index.md]: You may need a blank line before a bulleted list for it to render properly; do you have a preview link for your change? See the top-level README for instructions.

**Comment by mbrukman** [comment]
Also a few more meta-comments:

* [add a `Signed-off-by` line to your commits](https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md#commit-changes-and-sign-the-developer-certificate-of-origin)
* create a feature branch rather than use your `master` branch to contribute; [here's why](https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md#create-a-new-branch)
* add meaningful commit messages so that it's easy to see what was done; "Update index.md" doesn't tell the reader what this change means

You should probably squash the commits, add the `Signed-off-by` tag to the resulting commit, and you can also update the commit message at the same time.

**Comment by sumalaika** [comment]
I will open a new pull request

**Pull request closed without merging**